src/argilla/client/feedback/schemas.py

```diff
@@ -168,10 +367,80 @@ def update_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
 
 class LabelQuestion(_LabelQuestion):
+    """A label question schema for a feedback dataset.
+
+    Args:
+        name: The name of the question.
+        title: The title of the question.
+        description: The description of the question.
+        required: Whether the question is required or not.
+        settings: The settings of the question.
+        labels: The labels of the label question.
+        visible_labels: The number of visible labels of the label question.
+
+    Examples:
+        >>> import argilla as rg
+        >>> question = rg.LabelQuestion(
+        ...     name="relevant",
+        ...     title="Is the response relevant for the given prompt?",
+        ...     description="Select all that apply",
+        ...     required=True,
+        ...     settings={"type": "label_selection"},
+        ...     labels=["Yes", "No"],
+        ...     visible_labels=None
+        ... )
+        >>> # or use a dict
+        >>> question = rg.LabelQuestion(
+        ...     name="relevant",
+        ...     title="Is the response relevant for the given prompt?",
+        ...     description="Select all that apply",
+        ...     required=True,
+        ...     settings={"type": "label_selection"},
+        ...     labels={"yes": "Yes", "no": "No"},
+        ...     visible_labels=None
+        ... )
+
+    """
+
     settings: Dict[str, Any] = Field({"type": "label_selection"})
 
 
 class MultiLabelQuestion(_LabelQuestion):
+    """A multi label question schema for a feedback dataset.
+
+    Args:
+        name: The name of the question.
+        title: The title of the question.
+        description: The description of the question.
+        required: Whether the question is required or not.
+        settings: The settings of the question.
```

Comment:
And I would remove the `settings` from everywhere, since the user is not able to modify them, so we can ignore them! Also in the next minor release we'll remove those, so either keep them as is and remove then in the next release, or remove them now to avoid confusion.